homework/insensitive-palindrom/palindrome.cpp

```diff
@@ -0,0 +1,20 @@
+#include <algorithm>
+#include <cctype>
+#include <iterator>
+#include <iostream>
+
+#include "palindrome.hpp"
+
+bool is_palindrome(const std::string& str) {
+
+	std::string cleaned_str{};
+	std::copy_if(str.begin(), str.end(), 
+		     std::back_inserter(cleaned_str),
+		     [](auto c) { return isalnum(c); });
+	std::transform(cleaned_str.begin(), cleaned_str.end(), 
```

Comment:
Or better here -> Use transform and erse -> iterator returned by transform will be last valid, and rest of it you can erase